tools/create_release.py

```diff
@@ -170,10 +178,12 @@ async def main():
 
     if platform.system() == "Windows":
         create_windows_zip(package_folder)
+        write_frozen_file_list(package_folder)
     elif platform.system() == "Darwin":
         create_macos_zip(app_folder)
```

Comment:
Any reason why this cant be done for macos?

Reply:
Funnily enough the feature itself is not enabled for linux, though I think it works fine there?

I have no clue how adding files to the .app works and I didn't want to look into it

Reply:
can this get some comment added then? going ahead and putting the function would also be an alternative, but i fear itd cause some whacky breakage somewhere

Reply:
actually, issue would work too